docs/2.8.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -194,6 +194,64 @@ The ping command creates two contracts between the admin parties, then exercises
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Upgrade to Release 2.8
+^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I'd mention the deprecation of protocol version 3 and 4. They are not deleted but we should already indicate that they will remove